kong/client.go

```diff
@@ -363,3 +363,37 @@ func (c *Client) RootJSON(ctx context.Context) ([]byte, error) {
 
 	return body, nil
 }
+
+// SendConfig sends out the specified config to configured Admin API endpoint
+func (c *Client) SendConfig(ctx context.Context, config Configuration) error {
+	type sendConfigParams struct {
+		CheckHash int `url:"check_hash"`
+	}
+	req, err := c.NewRequest("POST", "/config", sendConfigParams{CheckHash: 1}, config)
```

Comment:
In the case of a public library, I think that would make sense to allow passing the `CheckHash` param to the method.